ts/Core/Axis/DateTimeAxis.ts

```diff
@@ -50,7 +51,7 @@ declare module './AxisOptions' {
 declare module '../Series/SeriesOptions' {
     interface SeriesOptions {
         pointInterval?: number;
-        pointIntervalUnit?: Highcharts.SeriesPointIntervalUnitValue;
+        pointIntervalUnit?: Series.PointIntervalUnitValue;
     }
```

Comment:
Please move type `PointIntervalUnitValue` to the class namespace of `DateTimeAxis`. It was a mistake to have it declared in the series file.